package.json

```diff
@@ -30,62 +30,64 @@
     "focus-visible": "^5.2.0"
   },
   "devDependencies": {
-    "@alaskaairux/design-tokens": "^3.12.2",
-    "@alaskaairux/icons": "^4.22.1",
+    "@alaskaairux/design-tokens": "^3.13.0",
+    "@alaskaairux/icons": "^4.27.0",
     "@alaskaairux/webcorestylesheets": "^3.7.3",
     "@aurodesignsystem/eslint-config": "^1.3.0",
-    "@babel/eslint-parser": "^7.21.3",
-    "@commitlint/cli": "^17.4.4",
-    "@commitlint/config-conventional": "^17.4.4",
-    "@open-wc/testing": "^3.1.7",
-    "@rollup/plugin-node-resolve": "^15.0.1",
-    "@semantic-release/changelog": "^6.0.2",
+    "@babel/eslint-parser": "^7.21.8",
```

Comment:
```suggestion

```

Please cross-reference between this package.json and the one in the Generator. 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json